packages/jsii-pacmak/lib/dependency-graph.ts

```diff
@@ -88,7 +90,12 @@ async function real$traverseDependencyGraph(
   ]);
   return Promise.all(
     Array.from(deps)
-      .filter((m) => !util.isBuiltinModule(m))
+      // No need to pacmak the dependency if it's built-in, or if it's bundled
+      .filter(
+        (m) =>
+          !util.isBuiltinModule(m) &&
+          !(meta.bundledDependencies ?? meta.bundleDependencies)?.includes(m),
```

Comment:
Unfortunately, the spec could be interpreted as implying that if both are provided, both will be honored (sigh).

```suggestion
          !meta.bundledDependencies?.includes(m) &&
          !meta.bundleDependencies?.includes(m),
```